acceptance/openstack/common.go

```diff
@@ -17,3 +20,21 @@ func PrintExtension(t *testing.T, extension *extensions.Extension) {
 	t.Logf("Updated: %s", extension.Updated)
 	t.Logf("Links: %v", extension.Links)
 }
+
+func DefaultSecurityGroup(t *testing.T, computeClient *golangsdk.ServiceClient) string {
```

Comment:
If it requires compute client, maybe create its instance inside of the function?

Reply:
solved